libcaliptra/inc/caliptra_types.h

```diff
@@ -254,6 +254,10 @@ struct dpe_resp_hdr {
 #endif
 
```

Comment:
General suggestion, the definitions that are coming from iROT profile spec being maintained in separate header file and being used in here.

Reply:
That's a decent idea. In general, libcaliptra as a whole needs more automated generation. The generation of most of the structs shouldn't be manual. I'd expect there would be a separation like you are describing if/when that finally gets addressed.